portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,29 +33,56 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-    private ArrayList<String> list;
+    private Comments list = new Comments();
 
     @Override
-    public void init(){
-        list = new ArrayList<String>();
-            list.add("Nice website");
-            list.add("I like the site");
-            list.add("good job"); 
-    }   
-
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    String json = convertToJsonUsingGson(list);
-    response.setContentType("application/json;");
-    response.getWriter().println(json);
-
-  }
-  private String convertToJsonUsingGson(ArrayList<String> list) {
-    Gson gson = new Gson();
-    String json = gson.toJson(list);
-    return json;
-  }
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        Query query = new Query("Comment");//.addSort(/**"timestamp",*/ SortDirection.DESCENDING);
+
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        PreparedQuery results = datastore.prepare(query);
+
+        List<String> comments = new ArrayList<>();
+        for (Entity entity : results.asIterable()) {           
+            String comment = (String) entity.getProperty("comment");
+            comments.add(comment);
+    }
```

Comment:
can you run an auto formatting tool? this brace is kind of messed up. i think that deigo or jeremy may know how?